### PR TITLE
Applying FakeMaven to SodgMojoTest (#1557)

### PR DESCRIPTION
**Pull request by PickBas**
Closes: #1557

### CODE REVIEW

**Comment by PickBas** [comment]
@mximp review please

File: eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java [hunk 1]
```diff
@@ -148,6 +148,21 @@ public FakeMaven withTojoAttribute(final String attribute, final Object value) {
         return this;
     }
 
+    /**
+     * Sets foreign with path.
+     *
+     * @param path Tojo attribute.
+     * @return The same maven instance.
+     */
+    public FakeMaven withForeignPath(final Path path) {
```
Comment (mximp) [inline eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java:157]: @PickBas this method is very similar to `withProgram(Path)`. Doesn't it do the job?

File: eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java [hunk 2]
```diff
@@ -204,6 +219,12 @@ public <T extends AbstractMojo> FakeMaven execute(final Class<T> mojo) throws IO
             "outputDir",
             this.workspace.absolute(Paths.get("target").resolve("classes")).toFile()
         );
+        final Path sodg = Paths.get("eo");
```
Comment (mximp) [inline eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java:222]: @PickBas Can be inlined as it's used only in single place

Comment (PickBas) [reply eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java]: Fixed this one

File: eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java (hunk 1 above)
Comment (PickBas) [reply eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java]: Removed. Check please, @mximp

**Review by mximp — APPROVED** [review]

**Comment by mximp** [comment]
@yegor256 please merge

**Comment by PickBas** [comment]
@mximp Have a look at the build failure please. Did this issue with fibonacci build appeared after my changes?

**Comment by mximp** [comment]
@PickBas I believe no.. There are some issues with the build at the moment.
You can try to build check it locally: `mvn clean install -Pqulice --errors`

**Comment by PickBas** [comment]
@mximp It builds locally without any problem

**Comment by PickBas** [comment]
@mximp Updated branch. Merge please

**Comment by mximp** [comment]
@yegor256 please try to merge again

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**